Formula/bind.rb

```diff
@@ -55,6 +60,7 @@ def install
 
     args = [
       "--prefix=#{prefix}",
+      "--sysconfdir=#{bindconfdir}",
```

Comment:
```suggestion
      "--sysconfdir=#{pkgetc}",
```

Reply:
Thanks for the suggestion to use "pkgetc". I would have used it if I had known about it, but it's not documented in the formula cookbook (I just checked again to make sure I hadn't missed it).

Reply:
The docs don't quite expose everything about the `Formula` class; that would be quite a lot. `pkgetc` is documented [here](https://rubydoc.brew.sh/Formula.html#pkgetc-instance_method).